contracts/libraries/EscrowState.sol

```diff
@@ -71,11 +71,16 @@ library EscrowState {
 
     /// @notice Initializes the Escrow state to SignallingEscrow.
     /// @param self The context of the Escrow State library.
-    /// @param minAssetsLockDuration The minimum assets lock duration.
-    function initialize(Context storage self, Duration minAssetsLockDuration) internal {
+    /// @param minAssetsLockDuration The initial minimum assets lock duration.
+    /// @param maxMinAssetsLockDuration Sanity check for max assets lock duration.
+    function initialize(
+        Context storage self,
+        Duration minAssetsLockDuration,
+        Duration maxMinAssetsLockDuration
+    ) internal {
         _checkState(self, State.NotInitialized);
         _setState(self, State.SignallingEscrow);
-        _setMinAssetsLockDuration(self, minAssetsLockDuration);
+        setMinAssetsLockDuration(self, minAssetsLockDuration, maxMinAssetsLockDuration);
```

Comment:
Let's also remove private method `_setMinAssetsLockDuration()` and move its logic into the `setMinAssetsLockDuration()`